openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,53 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(TimeStampedModel):
+    """
+    Model to save invitations, sent out to users for becoming admin of business unit(s).
+    """
+
+    name = models.CharField(verbose_name=_('Name'), max_length=255, null=True, blank=True, )
+    email = models.EmailField(verbose_name=_('Email'), max_length=255, unique=True, null=False, blank=False, )
+    business_unit = models.ForeignKey(
+        BusinessLine,
+        verbose_name=_('Business Unit'),
+        related_name='invitation',
```

Comment:
Related name is usually set as plural, as a foreign key relationship is many to one relationship and a single business unit can have many invitations associated with it. So please update the related_name to 'invitations'.